Artifacts/MsSql/Structure/Security/0040-Tables - RLPhase2.sql

```diff
@@ -0,0 +1,151 @@
+-- SPDX-License-Identifier: Apache-2.0
```

Comment:
@vimayya We discussed renaming these scripts previously, right? Do you know where we landed on the names?

Reply:
updated with 0040-Tables-MultipleAuthStrats.sql